content/english/vegan-culture/movies-worth-watching.md

```diff
@@ -0,0 +1,79 @@
+**links:** [Youtube trailer](https://www.youtube.com/watch?v=AjCebKn4iic), [Netflix](https://www.netflix.com/nz/title/80091936). 
+
+## Dark City
+**By:** Alex Proyas (Director & Writer)    
+**Country or Origin:**  United States   
+**Genre:** Mystery, Sci-fi, Thriller.   
+**Release Date:** 1998   
+**Issues covered:**  Vivisection, animal rights, [human superiority]({{< ref "/anti-vegan/humans-are-superior" >}}).  
+**Thoughts:** A dark, brooding, Cartesian nightmare where everything is filtered through a veil of doubt. Primarily aimed at the philosophical question 'are we more than the sum of our memories'. However, Dark City is also a useful thought experiment on vivisection and human justification for vivisection. *Gregory Sadler* notes that the experimenters are “'antagonists' rather than 'villains'. There is a reason for this – despite appearances, they are not actually evil, merely misguided. They have what seem to them perfectly good reasons for what they do, and they seem genuinely frightened and disturbed when their plans are disrupted... *- ([Dark City, Descartes, and Narrative Doubts](http://web.archive.org/web/20151002092115/https://gbsadler.blogspot.com/2011/09/dark-city-descartes-and-narrative.html))*.  Pay close attention to the human doctor (especially the mouse in the maze) and the film's final scenes.     
+**links:**  [Youtube trailer](https://www.youtube.com/watch?v=jSpowoKqSzc), [Amazon DVD](http://web.archive.org/web/20180504113443/https://www.amazon.com/Dark-City-Rufus-Sewell/dp/0780622553), [Prime Video](https://www.amazon.com/Dark-City-Rufus-Sewell/dp/B00IK42NDUmighty ape).
+
+## Planet of the Apes (PotA) - Prequel Trilogy 
+### The prequel trilogy: Rise of PotA, Dawn of PotA, War for PotA.
+**By:** Rupert Wyatt (Director for Rise) Matt Reeves (Director for Dawn & War)  
+**Country or Origin:** United States   
+**Genre:** Action, Drana, Sci-Fi   
```

Comment:
Drama

Reply:
Drana - the great new genre :)

Otherwise = Done